vertx-web/src/main/java/io/vertx/ext/web/impl/RoutingContextImpl.java

```diff
@@ -40,6 +40,8 @@
  */
 public class RoutingContextImpl extends RoutingContextImplBase {
 
+  private static final String SESSION_USER_HOLDER_KEY = "__vertx.userHolder";
```

Comment:
We should not add this, instead refer to the same constant declared on the session handler to keep all references right.

Reply:
Should I make that public? It doesn't seem to be in the same package and wasn't sure how else to keep it private.

Reply:
I made those 3 session key defines public.